### PR TITLE
feat: add jq and curl to frequency docker images

### LINKED ISSUE #1865: Add curl and jq to Frequency Docker images
As someone using a Frequency Docker image, I want to be able to implement specific heathchecks. To do so using the docker healthcheck system, the tools of `curl` and `jq`, while not absolute, are most common and easy to use.

Note this could be done in conjunction with #1390, although this would be good for others to implement stronger or different health checks than the default.

### PR DESCRIPTION
**Pull request by JoeCap08055**
# Goal
The goal of this PR is to add the `jq` and `curl` binaries to our published Docker images for Frequency.

Closes #1865 

# Discussion

# Checklist
- [ ] Chain spec updated
- [ ] Custom RPC OR Runtime API added/changed? Updated js/api-augment.
- [ ] Design doc(s) updated
- [ ] Tests added
- [ ] Benchmarks added
- [ ] Weights updated

### CODE REVIEW

**Review by saraswatpuneet — APPROVED** [review]

**Review by enddynayn — APPROVED** [review]
⚡ nice!

File: docker/instant-seal-node.dockerfile [hunk 1]
```diff
@@ -7,11 +7,15 @@ FROM --platform=linux/amd64 ubuntu:22.04 AS base
...
 
-RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates
+RUN apt-get update && apt-get install -y ca-certificates jq curl && update-ca-certificates
 
 # This is the 2nd stage: a very small image where we copy the Frequency binary
 FROM --platform=linux/amd64 ubuntu:22.04
 
+# We want jq and curl in the final image, but we don't need the support files
+RUN apt-get update && apt-get install -y jq curl && apt-get clean
+RUN rm -rf /usr/share/doc /usr/share/man /usr/share/zsh
```
Comment (wilwade) [inline docker/instant-seal-node.dockerfile:17]: These would need to be one line in order not to make a layer without those additional files.

I think without it being one line, the image size will not change

**Review by wilwade — CHANGES_REQUESTED** [review]
One potentially blocking comment

File: docker/instant-seal-node.dockerfile (hunk 1 above)
Comment (JoeCap08055) [reply docker/instant-seal-node.dockerfile]: Fixed.

**Review by aramikm — APPROVED** [review]
Ship it!

**Review by wilwade — APPROVED** [review]
👍

**Review by mattheworris — APPROVED** [review]
🚀

**Review by claireolmstead — APPROVED** [review]
Seems straight forward 👍🏼

**Pull request merged**